libs/ledger-live-common/src/api/Manager.ts

```diff
@@ -571,6 +571,28 @@ const installMcu = (
   }).pipe(remapSocketError(context));
 };
 
+function retrieveMcuVersion(
+  finalFirmware: FinalFirmware
+): Promise<McuVersion | undefined> {
+  return getMcus()
+    .then((mcus) =>
+      mcus.filter((deviceInfo) => {
+        const provider = getProviderId(deviceInfo);
+        return (mcu) => mcu.providers.includes(provider);
```

Comment:
mcu has an `any` type - but `getMcus` returns a `Promise<any>` because it's a request .. In the future we should maybe implement a type system for the manager

Reply:
Yeah, this has been directly grabbed from legacy code. So I don't fully know what the types are. It will require a bit of reverse engineering of the endpoint. I'll add a TODO